### ISSUE TITLE
Update gemsepc to work with Rails 3.1

### ISSUE THREAD

**Issue by aaronchi** [Sutto/slugged#10]
:)

**Comment by jhender** [comment]
same issue here. can it be used with 3.1? 
gemfile dependency doesn't allow at the moment

**Comment by aaronchi** [comment]
you may want to look at friendly_id v4 beta. It's got most of the same functionality

**Comment by jhender** [comment]
thanks aaronchi. i have already tried the other fork and seems to install fine

**Comment by hardfocus** [comment]
$ bundle install
Fetching source index for http://rubygems.org/
Bundler could not find compatible versions for gem "activerecord":
  In Gemfile:
    slugged (>= 0) ruby depends on
      activerecord (~> 3.0.0) ruby

```
rails (= 3.1.3) ruby depends on
  activerecord (3.1.3)
```

**Comment by minaguib** [comment]
Same problem with rails 3.2:

$ bundle
Fetching source index for https://rubygems.org/  
Bundler could not find compatible versions for gem "activesupport":
  In Gemfile:
    slugged (>= 0) ruby depends on
      activesupport (~> 3.0.0) ruby

```
rails (= 3.2.0) ruby depends on
  activesupport (3.2.0)
```

**Comment by hardfocus** [comment]
I ended up using https://github.com/norman/friendly_id instead.... Does the job just fine!

**Comment by Sutto** [comment]
Hey guys, Sorry I'd left this so long - it's been fixed for a few weeks now, and I've just set it up with travis and release 1.1.0 which includes that and fixes for Rails 3.2.3.

Once again, apologies for the issues. Hopefully it should be better from now on.
